daemon/api_themes.go

```diff
@@ -48,6 +48,12 @@ var (
 		ReadAccess:  themesOpenAccess{},
 		WriteAccess: themesAuthenticatedAccess{Polkit: polkitActionManage},
 	}
+
+	themesChangeCmd = &Command{
```

Comment:
it would be a bit better if this had a name that matched the url and possibly be defined in a api_accessories.go file?